components/Menu.js

```diff
@@ -1,56 +1,63 @@
-import React, { useState } from 'react'
-import Image from 'next/image'
-import styles from '../styles/Menu.module.css'
-import { Socials } from '../components/Socials'
-import { Pages } from './Pages'
+import React, { useState } from "react"
+
+import Image from "next/image"
+import Link from "next/link"
```

Comment:
Good call had been meaning to do this, worked up the template in plain HTML/CSS before moving it into Next